### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by DivyanshSareen**
Note: peace.css is only for testing purposes, will add imports for all the components once p1 components are done

### CODE REVIEW

**Review by abhishektanwar — APPROVED** [review]
Good to merge .

**Review by archit7448 — APPROVED** [review]
looks good to merge , proper naming and comments

**Review by bhushannaikaroskar — COMMENTED** [review]
Rest all I think its good to merge. Just one suggestion use shorter comments and if possible avoid them at first place.

File: style.css [hunk 1]
```diff
@@ -1,38 +1,86 @@
...
+
 .side-logo img{
     height: 100%;
 }
 
 .side-item{
-    box-shadow: unset;
     font-weight: var(--semi-bold);
     font-size: 0.9rem;
     color: var(--primary-900);
```
Comment (bhushannaikaroskar) [inline style.css:78]: please convert this to px

**Pull request merged**